src/interfaces/filing-interfaces/chg-registration-interfaces.ts

```diff
@@ -9,7 +10,7 @@ export interface ChgRegistrationIF {
     identifier: string
     naics?: NaicsIF
   }
-  courtOrder?: string
+  courtOrder?: CourtOrderIF
```

Comment:
Old code was incorrect. This new code matches [the schema](https://github.com/bcgov/business-schemas/blob/87117200269bc8ca56659c43b2cf5c1610f2f06b/src/registry_schemas/schemas/change_of_registration.json#L46).